handsontable/src/plugins/dropdownMenu/dropdownMenu.js

```diff
@@ -350,7 +351,9 @@ export class DropdownMenu extends BasePlugin {
    * @fires Hooks#afterDropdownMenuShow
    */
   open(position, offset = { above: 0, below: 0, left: 0, right: 0 }) {
-    if (this.menu?.isOpened()) {
+    const dialogPlugin = this.hot.getPlugin('dialog');
```

Comment:
The same as for the context menu case. The dropdown menu is not aware of the dialog plugin.